WinUIGallery/TabViewPages/TabViewWindowingSamplePage.xaml.cs

```diff
@@ -69,19 +69,6 @@ void SetupWindow()
             }
 
             Tabs.SelectedIndex = 0;
-
-
-#if UNIVERSAL
-            // Extend into the titlebar
-            var coreTitleBar = CoreApplication.GetCurrentView().TitleBar;
-            coreTitleBar.ExtendViewIntoTitleBar = true;
-
-            coreTitleBar.LayoutMetricsChanged += CoreTitleBar_LayoutMetricsChanged;
-
-            var titleBar = Windows.UI.ViewManagement.ApplicationView.GetForCurrentView().TitleBar;
-            titleBar.ButtonBackgroundColor = Microsoft.UI.Colors.Transparent;
-            titleBar.ButtonInactiveBackgroundColor = Microsoft.UI.Colors.Transparent;
-#endif
```

Comment:
nit : there is a scope to improve this sample to use winui 3 custom titlebar but this is good enough for now

Reply:
I'm going to keep the scope of this change as-is. Both comments can be addressed separately.